README.md

```diff
@@ -69,37 +77,15 @@ This section will list any kind of tools that will help other people to access t
 
 * [Online Compiler](https://play.google.com/store/apps/details?id=app.compiler)
 
-## Development Status and Ideas
-
-### Development Status
```

Comment:
this should go on the projects or road map.  I think I will save a copy of this.

Reply:
I remove the Idea part because an improved version of this exist on git ecosystem which is creating an issue explaining how this idea improves the project. Check #12 

About Development Status we can do the same, we can select some issues posted and work on them in a period of time and mesure that as Development status.

So the next step would be create those issues and determine a period to have them done.